_data/simple-icons.json

```diff
@@ -6228,7 +6231,8 @@
         {
             "title": "Nutanix",
             "hex": "024DA1",
-            "source": "https://www.nutanix.com/content/dam/nutanix/en/cmn/documents/nutanix-brandbook.pdf"
+            "source": "https://www.nutanix.com/",
+            "guidelines": "https://www.nutanix.com/content/dam/nutanix/en/cmn/documents/nutanix-brandbook.pdf"
```

Comment:
Even for guidelines, it would be preferable to find a source that is not a direct link to a file (for the same reason)... In this case, we might make an exception since I cannot find anywhere this file is linked...

Reply:
many of these files are not publicly linked.

Reply:
because they show up un google image searches.

Reply:
Google _image_ searches or Google searches?

In either case, Google **probably** only found those PDFs because they're linked publicly, because that's how crawlers work generally. So they _should_ be linked publicly somewhere. Not saying it's easy to find though.

Anyway, I think it's fine to make an exception for the guidelines.

Reply:
> Google _image_ searches or Google searches?
> 
> In either case, Google **probably** only found those PDFs because they're linked publicly, because that's how crawlers work generally. So they _should_ be linked publicly somewhere. Not saying it's easy to find though.
> 
> Anyway, I think it's fine to make an exception for the guidelines.

both searches. PDFs like this can show up in either.